### PR TITLE
webapp/terminal: fixing hidden scrollbar

### PR DESCRIPTION
**Pull request by haraldschilly**
# Description
This brings the div with the scrollbar in front of the other one: #3389

There is also some artifact left from exposing the render type … but it's no good to have that context menu even in dom mode.

# Testing Steps
I tested this with a bright (black on white) and a dark theme, in chrome and firefox, and also when I've both opened at the same time. Didn't notice any Problems.

# Relevant Issues
 #3389

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
- [ ] No debugging console.log messages.
- [ ] All new code is actually used.
- [ ] Non-obvious code has some sort of comments.

Front end:
- [ ] Restart at least one project and check its `~/.smc/local_hub/local_hub.log`
- [ ] Completely restart Webpack with `./w` in `/src`
- [ ] Screenshots if relevant.

### CODE REVIEW

File: src/smc-webapp/frame-editors/terminal-editor/connected-terminal.ts [hunk 1]
```diff
@@ -118,8 +118,12 @@ export class Terminal {
     this.set_connection_status("disconnected");
   }
 
+  static get rendererType() {
+    return "dom";
+  }
+
   private get_xtermjs_options(): any {
-    const rendererType = "dom";
+    const rendererType = Terminal.rendererType;
```
Comment (williamstein) [inline src/smc-webapp/frame-editors/terminal-editor/connected-terminal.ts]: Why did you change this to a computed getter?   I can't see any gain, except to make things more complicated and harder to understand.  Also, the code is now inconsistent in style to the code for `const fontFamily = "monospace";` directly below.   

The only other renderer is canvas, and it is unusably slow to create new Terminals, so we will likely never use it.

Comment (williamstein) [reply src/smc-webapp/frame-editors/terminal-editor/connected-terminal.ts]: Can you address whether this fix should really be upstream (in your opinion)?

Comment (haraldschilly) [reply src/smc-webapp/frame-editors/terminal-editor/connected-terminal.ts]: well, I just wanted this to be read-only attribute... there is no `const this.attr = ...`

**Comment by williamstein** [comment]
Please read about readonly and private here then change this code...

https://www.typescriptlang.org/docs/handbook/classes.html

**Comment by haraldschilly** [comment]
I changed it to a readonly attribute.

**Pull request merged**